### PR TITLE
Save pdfjs annotation to back-end - MEDT-3938

### PR DESCRIPTION
**Pull request by nikolas**
* Fix page number bug
* Create pdf annotation on save

### CODE REVIEW

File: mediathread/djangosherd/models.py [hunk 1]
```diff
@@ -357,6 +357,14 @@ def __str__(self):
                                              self.asset.title,
                                              self.asset.course.title)
 
+    def is_global_annotation(self):
+        # Disable "global annotation" stuff for PDF type. Let's phase
```
Comment (sdreher) [inline mediathread/djangosherd/models.py:361]: I would *so enjoy* removing global annotations from the codebase.

File: mediathread/assetmgr/models.py [hunk 2]
```diff
@@ -270,6 +270,12 @@ def global_annotation(self, user, auto_create=True):
 
     def media_type(self):
         label = 'video'
+
+        try:
```
Comment (sdreher) [inline mediathread/assetmgr/models.py:274]: Is there a situation where primary does not exist?

Comment (nikolas) [reply mediathread/assetmgr/models.py:274]: During testing, with the new is_global_annotation() change, some of the reports tests fail with Source.DoesNotExist. Debugging this was pretty confusing - we can possibly solve this another way, or postpone the global annotation changes altogether by just using the hardcoded ranges in the new PDF annotation structure.

Comment (sdreher) [reply mediathread/assetmgr/models.py:274]: I like your suggestion of just using hard-coded values in the ranges. To me, that feels cleaner then adding additional exception code. What's your feeling?

Comment (nikolas) [reply mediathread/assetmgr/models.py:274]: Sure, that's fine - i mean that's what we're currently doing for image annotations. That's what I was trying to get rid of in the first place, but I can just do that for now if we don't want to spend time re-thinking global annotations yet.

Comment (sdreher) [reply mediathread/assetmgr/models.py:274]: Yeah - removing one workaround for images/pdf seems simpler than removing two. That said, I'm fine talking through how we can remove global annotations now. No reason to put that off.

**Pull request merged**